platforms/mqtt/mqtt_adaptor.go

```diff
@@ -3,17 +3,17 @@ package mqtt
 import (
 	"crypto/tls"
 	"crypto/x509"
+	"fmt"
 	"os"
 
 	"gobot.io/x/gobot/v2"
 
 	paho "github.com/eclipse/paho.mqtt.golang"
-	"github.com/pkg/errors"
```

Comment:
Can't it be just "errors"?

Reply:
yes, this would be also possible, but a good practice is to use one of "fmt.Errorf()" or "errors.New()" in one file (some say also in one project)